defender/src/update.ts

```diff
@@ -0,0 +1,47 @@
+import { AutotaskClient } from "defender-autotask-client";
+import { SentinelClient } from "defender-sentinel-client";
+import config from "./config.js";
+import { decodeName } from "./networks.js";
+import { abi } from "./HypercertMinterABI.js";
+
+export const updateAutotask = async () => {
+  const autotaskClient = new AutotaskClient(config.credentials);
+
+  // Remove all old auto tasks and sentinels
+  const oldAutoTasks = await autotaskClient.list();
+  return await Promise.all([
+    ...oldAutoTasks.items.map((x) => {
```

Comment:
nit: for any non-trivial (more than a single line) anonymous function I generally prefer a more verbose variable name than something like `x`

Reply:
Fixed in latest commit